internal/stack/serverlessresources.go

```diff
@@ -88,12 +89,11 @@ func applyServerlessResources(profile *profile.Profile, stackVersion string, con
 // esHostWithPort checks if the es host has a port already added in the string , else adds 443
 // This is to mitigate a known issue in logstash - https://www.elastic.co/guide/en/logstash/current/plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-serverless
 func esHostWithPort(host string) string {
-	// The ES host of the form https://esHost or https://esHost:port
-	splitHost := strings.Split(host, ":")
+	url, _ := url.Parse(host)
```

Comment:
Should we handle the error here just in case?

Reply:
Thought so too. But not sure if I should return the `original host` back or a `nil` ?

Reply:
If this is not a valid URL we should probaby return an error. If we want to keep it simple for the caller we should return something potentially valid, so it would be better to return the original host, yes.

Lets avoid returning nil or an empty string unless we also return an error.